YarnSpinner/Value.cs

```diff
@@ -0,0 +1,351 @@
+
+using System;
+
+namespace Yarn
+{
+	// A value from inside Yarn.
+	public class Value : IComparable, IComparable<Value> {
+		public static readonly Value DEFAULT = new Value();
```

Comment:
@desplesda Love it, that fix is in! I wasn't able to remove the default constructor, however, as it's still needed to create Type.Variable Value's, as seen in `Parser.cs:694`. Any guidance you could provide would be much appreciated.